### DISCUSSION TITLE
Where do I get JAR file for Snowflake?

### DISCUSSION THREAD

**Discussion by moseleyi** [nielsbasjes/yauaa#690] [Q&A] [answered]
Hi,

Didn't know how else I could ask this question but the link in Snowflake documentation doesn't work

**Comment by nielsbasjes** [comment, marked as answer]
On what precise page (URL) is the download link incorrect?


The download link this page https://yauaa.basjes.nl/udf/snowflake/ is correct.

There is a link to "maven central":
https://search.maven.org/artifact/nl.basjes.parse.useragent/yauaa-snowflake/7.3.0/jar

Which shows you all the relevant download options.
Including a manual download (top right: Browse/Download)
Make sure you get the 'udf' jar file.

**Comment by moseleyi** [comment]
Thanks @nielsbasjes. I managed to get through the whole process and install the UDF :)
